README.md

```diff
@@ -8,10 +8,15 @@ The Scorecards GitHub Action is free for all public repositories. Private reposi
 
 ## Breaking changes in v2
 
-Starting from scorecard-action:v2, `GITHUB_TOKEN` permissions needs to incude
-`token_id: write` for `publish_results: true`. This is needed to access GitHub's
-OIDC token whuch verifies the authenticity of the result when publishing it.
-
+Starting from scorecard-action:v2, `GITHUB_TOKEN` permissions or job permissions needs to incude
+`id-token: write` for `publish_results: true`. This is needed to access GitHub's
+OIDC token which verifies the authenticity of the result when publishing it.
+
+scorecard-action:v2 also requires that the steps in the job running ossf/scorecard-action step
```

Comment:
hopefully that is the general case, since the example workflows do just that.

Reply:
Yeah hope so too. Would be good to make this generic enough and fix it in the right way though in the long run.